spec/mailers/course_update_email_mailer_spec.rb

```diff
@@ -0,0 +1,49 @@
+require "rails_helper"
+
+describe CourseUpdateEmailMailer, type: :mailer do
+  let(:course) { create(:course, :with_accrediting_provider) }
+  let(:mail) { described_class.send_course_update_email(course, "name") }
+
+  before do
+    course
+    mail
+  end
+
+  context "Sending an email to a user" do
+    it "Sends an email with the correct template" do
```

Comment:
Kind of nit-picky but think the convention is to down-case the first character of the string (`context` & `it`)